internal/powerplatform/resource_tenant_settings_test.go

```diff
@@ -146,6 +146,11 @@ func TestUnitTestTenantSettingsResource_Validate_Create(t *testing.T) {
 	httpmock.Activate()
```

Comment:
we might need additional unit/acc tests to explicitly validate new restoring tenant settings' values functionality and how zero_uuid work.
Or this is covered by env group set of tests?

Reply:
Will add after we merge the acc testing branch